### PR TITLE
Add raytune examples / tests

### PR DESCRIPTION
**Pull request by raubitsj**
Description
-----------
Add examples / tests for ray/tune

Also:
- Updated coveragetool to detect .coveragerc issues (found one, imports was there and didnt need to be)
- Updated .coveragerc to be sorted, because sorted is better

Requires:
- https://github.com/wandb/yea-wandb/pull/90

Possible future work:
- add more asserts to some of the tests, a little tedious, but should be done
- debug or document that ray1.12 not supported with wandb 1.13.0+ or workaround is needed (disable service)? (see t5_ test in ray)

Testing
-------
These are tests.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,15 +5,15 @@ envlist = black,
...
+          func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,ray112,ray2,service,docs,
                   imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc}-py37,
           standalone-{cpu,gpu,tpu,local}-py38,
           func-cover,
           cover
 
 [base]
 setenv =
-    YEA_WANDB_VERSION = 0.8.6
+    YEA_WANDB_VERSION = 0.8.7
```
Comment (dmitryduev) [inline tox.ini:16]: This will have to be 0.8.8, we released it yesterday for the unmock-server.

Comment (dmitryduev) [reply tox.ini:16]: (also, pinning flask<2.2.2 is not necessary, we have a hack that overcomes it -- see my comment to your yea-wandb PR)

**Review by dmitryduev — APPROVED** [review]
LGTM!

File: tox.ini (hunk 1 above)
Comment (raubitsj) [reply tox.ini]: reverted

**Pull request merged**